pkg/scalers/rabbitmq_scaler.go

```diff
@@ -41,6 +41,7 @@ const (
 	rabbitMetricType                       = "External"
 	rabbitRootVhostPath                    = "/%2F"
 	rmqTLSEnable                           = "enable"
+	connectionName                         = "keda-rabbitmq-external-scaler"
```

Comment:
I think that just setting `keda` is better. In the future we could support setting it by users
```suggestion
	connectionName                         = "keda"
```